### DISCUSSION TITLE
How to specify root directory to search from

### DISCUSSION THREAD

**Discussion by wolfisdev** [junegunn/fzf#2428] [Q&A]
I can't seem to figure out how I can tell fzf to start searching from another directory than the ```$PWD```.

I have this alias
```
alias pdfind='fzf --preview-window=hidden -i -e -q ".pdf$ " | xargs -I {} okular {}'
```
And I'd like it configure it to start searching from either ```/``` or ```$HOME```.

I'm sure there is an easy way to achieve this, but I couldn't find anything in the wiki or the man page.

Cheers.

**Comment by junegunn** [comment]
See https://github.com/junegunn/fzf#usage

**Reply by wolfisdev** [reply]
Apparently I didn't fully understand the way fzf is supposed to be used. I'm now piping `fd` output into fzf and provide the directory to `fd`.

**Reply by DinoChiesa** [reply]
@junegunn thanks for this wonderful , useful tool. 

the link provided in this answer does not provide "usage". The document referenced there says I can use options like `--walker` and `--walker-root` but _it does not describe those options, or their effects._

I needed to run `fzf --help` to understand that `--walker-root` specifies the directory from which fzf will search. The default is `.`. 
I didn't see that in the fzf README on github.  It's provided by the help command. 
![image](https://github.com/user-attachments/assets/ad709e4f-5667-4404-8840-6e5b513e182b)

**Reply by junegunn** [reply]
The README page is not supposed to be a comprehensive reference for fzf. [`man fzf`](https://github.com/junegunn/fzf?tab=readme-ov-file#options) is. Listing all options of fzf on this single page would make it too long and intimidating for the new users.

**Reply by John-Dennehy** [reply]
As an intimidated new user with the same question, I feel the README page doesn't really help. It is far too full of jargon I dont yet understand.

**Reply by junegunn** [reply]
fzf acts as a glue between various tools, so the user is expected to have some level of knowledge on shell scripting, but still, we try to make the doc less intimidating. Can you give some examples of "jargons" you were unable to understand? I'll see if it's possible to make them less intimidating without losing the necessary technical detail.
